apps/researcher/src/lib/user/actions.ts

```diff
@@ -0,0 +1,24 @@
+import {clerkClient} from '@clerk/nextjs';
+import {createPersistentIri} from '@colonial-collections/iris';
+import {unstable_noStore as noStore} from 'next/cache';
+
+export async function getOrSetIri(userId: string) {
```

Comment:
Minor: rename the function to make its meaning clearer? For example: `getIriOfUser()`?

We could also make clear in the name that the IRI is stored if it doesn't exist, but we can also regard this as an implementation details/part of the inner working of the function.